azure-pipelines.yml

```diff
@@ -55,23 +55,6 @@ stages:
   variables:
     AZURE_CI: 'true'
   jobs:
-  - job: prerelease_deps_coverage_64bit_blas
-    timeoutInMinutes: 90
-    pool:
-      vmImage: 'ubuntu-20.04'
-    steps:
-    - task: UsePythonVersion@0
-      inputs:
-        versionSpec: '3.9'
-        addToPath: true
-        architecture: 'x64'
-    - template: ci/azure-travis-template.yaml
-      parameters:
-        test_mode: full
-        coverage: true
-        numpy_spec: "--pre --upgrade --timeout=60 -i $PRE_WHEELS numpy"
-        other_spec: "--pre --upgrade --timeout=60"
-        use_64bit_blas: true
```

Comment:
@tupui 

> My understanding was that ILP64 was just for the windows job. I don't see this elsewhere in the Azure config.

This `use_64bit_blas` variable is where you are losing the testing of 64-bit integers by switching over to meson on Linux. It triggers some stuff in `ci/azure-travis-template.yaml` to pull in the the special OpenBLAS.

The job doesn't *always* time out on `main` as is--shall we try a PR that just adds your test skip before we branch `1.9.0` but defer the switch away from 64-bit integer CI testing until after branching if the job still doesn't behave (and perhaps open an issue to remind us to add it back once meson supports that, if no such issue exists already?)